Src/IMX/Api/src/Imx.Sdk.Gen/Imx.Sdk.Gen.csproj

```diff
@@ -29,7 +25,6 @@
     <PackageReference Include="RestSharp" Version="108.0.2" />
     <PackageReference Include="Polly" Version="7.2.3" />
     <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
-    <None Include="..\..\..\..\..\README.md" Pack="true" PackagePath="\"/>
-    <None Include="..\..\..\..\..\LICENSE" Pack="true" PackagePath="\"/>
```

Comment:
These will need to back added back in?
`    <None Include="..\..\..\..\..\README.md" Pack="true" PackagePath="\"/>
    <None Include="..\..\..\..\..\LICENSE" Pack="true" PackagePath="\"/>
`